net/mwan3/files/usr/libexec/rpcd/mwan3

```diff
@@ -111,6 +111,12 @@ get_mwan3_status() {
 		network_get_uptime uptime "$iface"
 		network_is_up "$iface" && up="1"
 
+		if [ -f "$MWAN3TRACK_STATUS_DIR/${iface}/STATUS" ]; then
+			status="$(cat "$MWAN3TRACK_STATUS_DIR/${iface}/STATUS")"
+		else
+			status="unknown"
```

Comment:
Can this instead read the interface status in `/var/run/mwan3/iface_status/${iface}` ?

The problem is that when tracking is disabled on an interface, `mwan3 status` will still report correctly but the ubus call will not.

Reply:
This has nothing to do with my change.
It's just an improvement.
If the status is unknown, because the file doesn't exist, then he should return `unknowon` and not just nothing.

I think the value you need is already present https://github.com/openwrt/packages/blob/master/net/mwan3/files/usr/libexec/rpcd/mwan3#L125